docs/index.rst

```diff
@@ -117,15 +117,22 @@ Nuts and bolts
    logging
    warnings
    utils/index
+
+{% if is_development %}
+
    testhelpers
    development/workflow/get_devel_version
 
+{%endif%}
+
 .. _developer-docs:
 
 ***********************
 Developer Documentation
 ***********************
 
+{% if is_development %}
+
 The developer documentation contains instructions for how to contribute to
```

Comment:
How about keeping this sentence which is the same outside of the `if` clause? Good to keep everything as much the same as possible.